docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:14.4
 
 # Required dependencies
-RUN apt-get update && apt-get install -yq python3-pip less && pip3 install boto3 && curl -fsSL --compressed  "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o /tmp/awscliv2.zip && unzip -q /tmp/awscliv2.zip -d /tmp && /tmp/aws/install && npm install serverless -g
+RUN apt-get update && apt-get install -yq python3-pip less && pip3 install boto3 && curl -fsSL --compressed  "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o /tmp/awscliv2.zip && unzip -q /tmp/awscliv2.zip -d /tmp && /tmp/aws/install && npm install serverless@2.64.1 -g
```

Comment:
loos like keeping global serverless here is needed since it runs the `serverless.sh` file

Maybe we should update `serverless.sh` to also not call global serverless. It's a bit of a chore since we also need to update the docs, so maaybe it's ok to do it later. AFAIK the `serverless.sh` file is not referenced anywhere else, so it should be safe to not update it right now

Reply:
sounds good I will leave it in for now